src/main/java/com/plaid/client/response/InvestmentsTransactionsGetResponse.java

```diff
@@ -0,0 +1,104 @@
+package com.plaid.client.response;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+/**
+ * Response object for /investments/transactions/get endpoint.
+ */
+public final class InvestmentsTransactionsGetResponse extends BaseResponse {
+  private ItemStatus item;
+  private List<Account> accounts;
+  private List<InvestmentTransaction> investmentTransactions;
+  private List<Security> securities;
+  private Integer totalInvestmentTransactions;
+
+  public ItemStatus getItem() {
+    return item;
+  }
+
+  public List<Account> getAccounts() {
+    return accounts;
+  }
+
+  public List<InvestmentTransaction> getInvestmentTransactions() {
+    return investmentTransactions;
+  }
+
+  public List<Security> getSecurities() {
+    return securities;
+  }
+
+  public Integer getTotalInvestmentTransactions() {
+    return totalInvestmentTransactions;
+  }
+
+  public static final class InvestmentTransaction {
+    private String investmentTransactionId;
+    private String accountId;
+    private String securityId;
+    private String date;
+    private String name;
+    private Double quantity;
+    private Double amount;
+    private Double price;
+    private Double fees;
+    @SerializedName("class") private String transactionClass;
```

Comment:
I think we're missing `subclass` here?

Reply:
we're not shipping `subclass` - removed it for GA